meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -1,2295 +1,45 @@
+    "BLEDisconnectLifecycleCoordinator",
+    "BLEConnectionOwnershipLifecycleCoordinator",
+    "BLEShutdownLifecycleCoordinator",
+    "_LifecycleStateAccess",
+    "_LifecycleThreadAccess",
+    "_LifecycleErrorAccess",
+    "_DisconnectPlan",
+    "_OwnershipSnapshot",
+    "_ORIGINAL_GET_CONNECTED_CLIENT_STATUS",
+    "_ORIGINAL_GET_CONNECTED_CLIENT_STATUS_LOCKED",
+    "_is_currently_connected_elsewhere",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)